avahi-monitor.py

```diff
@@ -47,16 +59,23 @@ def __init__(self):
 
         sda.start()
         self.sda = sda
+        self._registery = dict()
 
     def service_added_cb(self, sda, interface, protocol, name, type, h_type, domain, host, aprotocol, address, port, txt, flags):
         ifname = sda.siocgifname(interface)
         print "Found Service data for service '%s' of type '%s' (%s) in domain '%s' on %s.%i:" % (name, h_type, type, domain, ifname, protocol)
-        command.addNextHop(faceURI, cost=defaults.RouteCost.LOCAL_NETWORK)
+        faceUri = face_uri_from_triplet(type, host, port)
+        check_call(["nfdc", "add-nexthop",
```

Comment:
Do you want to handle `CalledProcessError` from this, or are you deliberately letting it propagate up to the caller?